tests/data/power_flow/pgm-automatic-tap/params.json

```diff
@@ -0,0 +1,15 @@
+{
+  "calculation_method": "newton_raphson",
+  "tap_changing_strategy": "any_valid_tap",
+  "rtol": 1e-5,
+  "atol": 1e-5,
+  "extra_params": {
+    "newton_raphson": {
+      "experimental_features": "enabled",
+      "fail": {
+        "reason": "Experimental feature under development",
+        "raises": "PowerGridError"
```

Comment:
is this really the most generic error message we get? 😮 it's fine like this, just surprised 😬

Reply:
Nope, we dont. I am going to add a `run` argument to xfail to avoid running this case in first place.

Reply:
what do you mean?